custom/icds_reports/ucr/data_sources/child_cases_monthly_v2.json

```diff
@@ -2184,8 +2207,68 @@
           "type": "property_name",
           "property_name": "no_exist"
         }
-      }
-    },
+      },
+      "zscore_grading_wfa": {
+        "type": "conditional",
```

Comment:
this expression does not seem correct. it will overwrite previous data to be `no_exist` once a new gmp date is changed. So the current month will show the zscore, and all previous months will show `no_exist`. Is that what you are intending?

Reply:
Hi @calellowitz,

Thanks for flagging this, you are right. This expression does not allow us to maintain historic data for the previous month and that is not what is needed. I am exploring an alternative for this.